style.css

```diff
@@ -320,3 +320,60 @@ section {
   display: flex;
   flex-direction: column;
 }
+
+.Contact-form {
+  background-color: #6070ff;
+  background-image: url("./images/headerBackgd.png");
+  background-size: cover;
+  border-radius: 60px 0 0 0;
+}
+
+form ul {
+  display: flex;
+  flex-direction: column;
+}
+
+#name,
+#mail,
+#textarea {
+  width: 350px;
+  border-radius: 7px;
+  border: none;
+  margin: 5px;
+  padding-left: 20px;
+}
+
+#textarea {
+  padding-top: 20px;
+}
+
+#name,
+#mail {
+  height: 40px;
+}
+
+form button {
+  width: 150px;
+  height: 50px;
+  border-radius: 7px;
+  border: none;
+}
```

Comment:
- [ ] Great job so far 👍, however, to be precise with the design, please add margin so that you will have space after the button.

### Your implementation
![image](https://user-images.githubusercontent.com/59416136/155173101-3c8219d3-7deb-40c2-be47-3496253514d1.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/155173210-702d1220-653a-4afe-bb7d-9c176a136a03.png)

**Kindly fix it.**